packages/api/src/graphql/invoice.ts

```diff
@@ -111,17 +116,21 @@ export const GraphQLInvoiceItemInput = new GraphQLInputObjectType({
     name: {type: GraphQLNonNull(GraphQLString)},
     description: {type: GraphQLString},
     quantity: {type: GraphQLNonNull(GraphQLInt)},
-    amount: {type: GraphQLNonNull(GraphQLInt)}
+    amount: {type: GraphQLNonNull(GraphQLInt)},
+    total: {type: GraphQLNonNull(GraphQLInt)},
```

Comment:
What is diff between amount and total.

Reply:
These values are part of the existing data model. Did not change that ;) I only had to complete the GQL response, so that I can display and update all information via user interface.